.husky/pre-commit

```diff
@@ -14,4 +12,4 @@ npm i --package-lock-only
 cd /$pwd
 npm run audit
 
-npx lint-staged
\ No newline at end of file
+lint-staged
```

Comment:
I think this still needs to be `npx lint-staged`. I ran this locally and got the error: 
".husky/pre-commit: 15: lint-staged: not found"